app/helpers/orders_helper.rb

```diff
@@ -38,13 +38,17 @@ def order_owner(owner_guid)
   end
 
   def contact_info(contact_address)
+    contact_address = {} if contact_address.nil?
+
     contact_address['Name'] = contact_name(contact_address)
     contact_address['Phone'] = contact_address.fetch('Phones', {}).fetch('Item', {})
 
     contact_address
   end
 
   def contact_name(contact)
-    [contact['FirstName'], contact['LastName']].reject(&:empty?).join(' ') unless contact.nil?
+    contact = {} if contact.nil?
```

Comment:
The desired result here is a string 'FirstName LastName' but if nothing is supplied we're returning an empty {} -- We should match the return types. Ultimately I'm not sure the nil check is necessary?